src/masternodes/rpc_proposals.cpp

```diff
@@ -650,55 +650,180 @@ UniValue listgovproposals(const JSONRPCRequest &request) {
         "listgovproposals",
         "\nReturns information about proposals.\n",
         {{"type", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "cfp/voc/all (default = all)"},
-          {"status", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "voting/rejected/completed/all (default = all)"}},
+          {"status", RPCArg::Type::STR, RPCArg::Optional::OMITTED, "voting/rejected/completed/all (default = all)"},
+          {"cycle",
+          RPCArg::Type::NUM,
+          RPCArg::Optional::OMITTED,
+          "cycle: 0 (all), cycle: N (show cycle N), cycle: -1 (show previous cycle) (default = 0)"}},
         RPCResult{"{id:{...},...}     (array) Json object with proposals information\n"},
```

Comment:
Let's add pagination as last parameter. 
Keeps it consistent with previous paginated RPCs and documents that it will be available via single object